src/scss/components/_newGameButton.scss

```diff
@@ -0,0 +1,45 @@
+.newGameButton {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+  align-items: center;
+  padding-right: 50px;
+  border: none;
+  -webkit-border-bottom-right-radius: 40px;
+  -moz-border-radius-bottomright: 40px;
+  border-bottom-right-radius: 40px;
+  background: rgba(255, 255, 255, 0.25);
+  color: rgb(255, 255, 255);
+  font-weight: bold;
+  font-size: 2rem;
+  letter-spacing: 0.1rem;
+  cursor: pointer;
+  transition: filter ease-in-out 0.2s, background-color ease-in-out 0.2s;
+
+  &:hover {
+    background: rgba(255, 255, 255, 0.35);
+    filter: saturate(1.25);
+  }
+  &__first-letter {
+    position: relative;
+    background: -webkit-linear-gradient(145deg, #35973e 0%, #063612 100%, #2c2c2c 90%);
+    background: -moz-linear-gradient(145deg, #35973e 0%, #063612 100%, #2c2c2c 90%);
+    background: -o-linear-gradient(145deg, #35973e 0%, #063612 100%, #2c2c2c 90%);
+    background: linear-gradient(145deg, #35973e 0%, #063612 100%, #2c2c2c 90%);
+    box-shadow: inset 0px 2px 2px rgba(75, 75, 75, 0.5);
+    border: 2px solid #063612;
+    padding: 10px 15px;
+    margin-right: 2px;
+  }
+
+  .newGameButton__first-letter > .elementNum {
```

Comment:
```suggestion
  .newGameButton__elementNum {
```

Reply:
Tak poprawnie z BEM

Reply:
Próbowałem tak tylko że w ten sposób to nie działa. Mogę zamienić na coś takiego ewentualnie `.newGameButton__first-letter .elementNum`